test/api/buffer/exports/update/expected-state.json

```diff
@@ -59,5 +59,43 @@
         }
       ]
     }
+  ],
+  "files": [
+    {
+      "name": "my_receiver_my_export_1_%s",
```

Comment:
Tu vidim 4 subory, aj v UI vidim 4 ... je tam este ten problem, ze by ti jeden chybal alebo bol naviac?

Reply:
Už není, nějak to samovolně zmizelo. 😅